packages/dx-react-grid/docs/guides/searching.md

```diff
@@ -29,6 +29,12 @@ In [controlled mode](controlled-and-uncontrolled-modes.md), pass the search opti
 
 .embedded-demo({ "path": "grid-searching/controlled-mode", "showThemeSelector": true })
 
+## Search in Visible Columns Only
+
+The Grid component searches all columns, including those that are hidden using the [`TableColumnVisibility`](../reference/table-column-visibility.md) plugin. To prevent searching invisible columns, use the `IntegratedFiltering` plugin's `columnExtensions` property.
```

Comment:
add a comma after "those that are hidden"

To prevent a search through invisible columns, use the...